stacks/level2_jobs_stack.py

```diff
@@ -0,0 +1,115 @@
+from typing import cast, Any
+from json import dumps
+
+from aws_cdk import Duration, Stack, RemovalPolicy
+from aws_cdk.aws_events import Rule, Schedule
+from aws_cdk.aws_events_targets import LambdaFunction
+from aws_cdk.aws_dynamodb import Table, Attribute, AttributeType
+from aws_cdk.aws_iam import Effect, PolicyStatement
+from aws_cdk.aws_lambda import Runtime
+from aws_cdk.aws_lambda_python_alpha import PythonFunction
+from aws_cdk.aws_sqs import Queue, DeadLetterQueue
+from constructs import Construct
+
+
+QUEUE_NAME = "OdinSMRLevel2Queue-{project}-{freqmode}"
+
+
+class Level2JobsStack(Stack):
+    def __init__(
+        self,
+        scope: Construct,
+        id: str,
+        projects: list[dict[str, Any]],
+        odin_api_root: str,
+        worker_key_ssm_name: str,
+        lambda_timeout: Duration = Duration.seconds(900),
+        lambda_schedule: Schedule = Schedule.rate(Duration.days(1)),
+        queue_retention_period: Duration = Duration.days(14),
+        **kwargs
+    ) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        lambda_name = "OdinSMRLevel2JobsLambda"
+        processing_table_name = "OdinSMRLevel2ProcessingTable"
+
+        level2_jobs_lambda = PythonFunction(
```

Comment:
I think you can replace to "Fuction" as it is. the runtime arg tells its python.

Reply:
Detta blir skillnaden:

``` python
-        level2_jobs_lambda = PythonFunction(
+        level2_jobs_lambda = Function(
             self,
             lambda_name,
-            entry="level2",
-            handler="handler",
-            index="handlers/level2_jobs.py",
+            code=InlineCode.from_asset("../level2/handlers"),
+            handler="level2_jobs.handler",
             runtime=Runtime.PYTHON_3_9,
             timeout=lambda_timeout,
             environment={
```